services/src/main/java/org/apache/unomi/services/impl/ParserHelper.java

```diff
@@ -145,5 +154,49 @@ public static void resolveValueType(DefinitionsService definitionsService, Prope
 
     interface ConditionVisitor {
         void visit(Condition condition);
+        void postVisit(Condition condition);
```

Comment:
the postVisit is never called, it should be called in the "visitConditions".

Reply:
Oops... sorry about that. Strange that the tests didn't catch that.